Refresh.GameServer/Endpoints/ApiV3/DataTypes/Response/ApiInstanceResponse.cs

```diff
@@ -39,8 +39,8 @@ public class ApiInstanceResponse : IApiResponse
     public required string SoftwareLicenseUrl { get; set; }
     
     public required bool RegistrationEnabled { get; set; }
-    public required AssetSafetyLevel MaximumAssetSafetyLevel { get; set; }
-    public required AssetSafetyLevel MaximumAssetSafetyLevelForTrustedUsers { get; set; }
+    public required AssetFlags BlockedAssetFlags { get; set; }
+    public required AssetFlags BlockedAssetFlagsForTrustedUsers { get; set; }
```

Comment:
i think this is the same argument as "values vs names" for enums, and we ended up landing on values there 
imo since this is an api response, it makes more sense to present it as a bitflag number, since that will save bandwidth and mean that if we change the names Refresh-side, it doesnt break the API

Reply:
Changing the names is a massive refactor that simply isn't going to happen. If the names do get changed/shuffled, it would have caused breaking changes both internally and in the config files anyways.

The bandwidth overhead is not going to matter.

Reply:
thats fair, i'll change it to be configassetflags